src/TockContext.tsx

```diff
@@ -157,10 +158,18 @@ const TockContext: (props: { children?: ReactNode }) => JSX.Element = ({
 }: {
   children?: ReactNode;
 }) => {
+  const retrieveUserId = () => {
```

Comment:
This function can be written outside component scope to avoid redeclaration.

Reply:
ok, I will extract it to utils.ts